<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="bookList">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>641</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFirst">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFront">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в начало</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBack">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в конец</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFront">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить в начале</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeBack">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить в конце</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeNum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить по номеру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchNum">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти по номеру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchAuth">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти по автору</string>
    </property>
   </widget>
   <widget class="QPushButton" name="isHere">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Узнать, есть ли книга в наличии</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>572</x>
      <y>300</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить в файл</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
